core/node/eth_sender/src/abstract_l1_interface.rs

```diff
@@ -50,9 +50,14 @@ pub(super) trait AbstractL1Interface: 'static + Sync + Send + fmt::Debug {
     async fn get_tx_status(
         &self,
         tx_hash: H256,
+        operator_type: OperatorType,
     ) -> Result<Option<ExecutedTxStatus>, EthSenderError>;
 
-    async fn send_raw_tx(&self, tx_bytes: RawTransactionBytes) -> EnrichedClientResult<H256>;
+    async fn send_raw_tx(
```

Comment:
new argument is added here to make tests easier to write, but overall this change would have been made when introducing gateway support anyway (gateway will be modelled as a new operator type)